app/views/courses/preview/_apply.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="govuk-!-margin-bottom-8">
+  <h2 class="govuk-heading-l" id="section-apply">Apply</h2>
+  <% if course.has_vacancies? %>
+    <p class="govuk-body">
+      <a href="https://2019.teachertraining.apply.ucas.com/apply/student/login.do">Apply on the UCAS website</a>. You’ll need to register with UCAS before you can apply.
```

Comment:
Could this be in Settings? It will change between recruitment cycles.

Reply:
I think this should probably be a call to a decorator or helper method which should return the right url based on the recruitment cycle that the course is in. Settings would be too static for this.